src/main/kotlin/com/exactpro/th2/rptdataprovider/producers/MessageProducer.kt

```diff
@@ -118,7 +155,7 @@ class MessageProducer(
         ).firstOrNull()?.let { if (it.isEmpty) null else it }
 
         return rawBatchNullable?.let { rawBatch ->
-            (codecCacheBatches.get(rawBatch.id.toString()) ?: fromRawMessage(rawBatch)).batch[id]
+            (codecCacheBatches.get(rawBatch.id.toString()) ?: parseRawMessageBatch(rawBatch, true)).batch[id]
```

Comment:
true is a default value for `needAttachEvents` parameter. Don't we need to use a default value in this case?